pom.xml

```diff
@@ -62,6 +62,8 @@
     </issueManagement>
 
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Can you change compiler version? Cube should be 1.7 compliant, not 1.8 for now :)